### ISSUE TITLE
Regular releases with updated ELI

### ISSUE THREAD

**Issue by rbuffat** [openstreetmap/iD#8466]
Currently, the imagery index is only updated with each release of iD.  This is ok when there are regular releases of iD., but the last version of iD was released on March 3th.  The latest commit to the develop branch was a month ago on March 18th, thus from the outside, it seems unlikely that there will be a new release soon. 

The last week without a commit to ELI was in September of last year. Personally, I perceive a duration of more than a month as too long for a change in ELI to propagate to the OSM homepage. 

I would suggest considering a weekly or bi-weekly patch release of iD with an updated version of ELI.

See also the discussion on openstreetmap-website:  https://github.com/openstreetmap/openstreetmap-website/issues/3179

**Comment by gy-mate** [comment]
Wouldn't #9730 be a better solution—one that doesn't depend on releases?

**Comment by mnalis** [comment]
> Wouldn't [#9730](https://github.com/openstreetmap/iD/issues/9730) be a better solution—one that doesn't depend on releases?

Given the comment in https://github.com/osmlab/editor-layer-index/issues/574#issuecomment-433628633, it doesn't look so:

> We will never take the autogenerated output on the master development branch of ELI and push it directly into iD. Never ever. Stop asking.

So perhaps regular iD releases are the easiest way to go.

**Comment by andrewharvey** [comment]
@mnalis you're referencing opinions from the previous maintainer of iD, this shouldn't limit us going forward.

I think regular releases of iD is a workaround, iD pulling the latest ELI on the fly from a URL that ELI publishes on a regular basis, seems like the way forward.

**Comment by mnalis** [comment]
> you're referencing opinions from the previous maintainer of iD, this shouldn't limit us going forward.

I agree it shouldn't, but _does_ it? Are you saying that iD's current maintainers (whoever they are) are in favor of that solution? Do you have pointers to it being actively discussed / worked on lately (i.e. in last month or three)?

https://github.com/openstreetmap/iD/issues/9730 seems like it has been implemented and working half a year ago, yet it still is not merged, nor do there seem to be further discussions on it. I've looked at several other attempts over time (but only looked for half an hour, so not exhaustively), but found none that look like they'll be implemented soon and where there is consensus about what should be done and how.
 
This one looks most promising in short term, and I didn't see anybody complain (yet) why it would be a bad idea.

Also note that regular iD release schedules (I'd even be content with monthly releases if (bi)weekly is too much):
- are **orthogonal to dynamic ELI** (i.e. implementing it does **not** preclude also doing dynamic ELI)
- has **other benefits** (i.e. incorporation of new/updated id-tagging-schema presets, NSI, translations, icons ...) _in addition_ to ELI. Sure, all of those could (perhaps) also be made dynamic, but that is likely a lot of work and might have other unintended consequences

**Comment by matkoniecz** [comment]
https://github.com/openstreetmap/iD/pull/12152

> blocked by either https://github.com/osmlab/editor-layer-index/pull/2927 or https://github.com/osmlab/editor-layer-index/pull/2530, depending on what option the ELI maintainers prefer.
